docs/tutorial/part-one/index.md

```diff
@@ -1,57 +1,57 @@
+![dự án Hello World trong VS Code](01-hello-world-vscode.png)
 
-_Note: Again, the editor shown here is Visual Studio Code. If you’re using a different editor, it will look a little different._
+_Lưu ý: Một lần nữa, Again, trình soạn thảo hiển thị ở đây là Visual Studio Code. Nếu bạn sử dụng một trình soạn thảo khác, nó sẽ trông hơi khác._
 
-Let’s take a look at the code that powers the homepage.
+Hãy cùng xem phần code tạo nên trang chủ.
 
-> 💡 If you stopped your development server after running `gatsby develop` in the previous section, start it up again now — time to make some changes to the hello-world site!
+> 💡 Nếu như bạn đã dừng máy chủ phát triển sau khi chạy lệnh `gatsby develop` in the previous section, hãy khởi động nó lại bây giờ, đã đến lúc để thực hiện một số thay đổi cho trang hello-world!
 
-## Familiarizing with Gatsby pages
+## Làm quen với các trang Gatsby
 
-Open up the `/src` directory in your code editor. Inside is a single directory: `/pages`.
+Mở thư mục `/src` trong trình soạn thảo code của bạn. Bên trong là một thư mục duy nhất: `/pages`.
 
-Open the file at `src/pages/index.js`. The code in this file creates a component that contains a single div and some text — appropriately, “Hello world!”
+Mở tệp tại `src/pages/index.js`.Phần code trong tệp này tạo ra một thành phần chứa một div và vài văn bản - thích hợp, “Hello world!”
 
-### ✋ Make changes to the “Hello World” homepage
+### ✋ Thay đổi trang chủ của "Hello World"
 
-1.  Change the “Hello World!” text to “Hello Gatsby!” and save the file. If your windows are side-by-side, you can see that your code and content changes are reflected almost instantly in the browser after you save the file.
+1.  Thay đổi văn bản "Hello World!" thành "Hello Gatsby!" và lưu tệp. Nếu các cửa sổ của bạn nằm cạnh nhau, bạn có thể thấy các thay đổi về code và nội dung được phản ánh gần như ngay lập tức trong trình duyệt sau khi bạn lưu tệp.
 
 <video controls="controls" autoplay="true" loop="true">
   <source type="video/mp4" src="./02-demo-hot-reloading.mp4"></source>
-  <p>Sorry! Your browser doesn't support this video.</p>
+  <p>Xin lỗi! Trình duyệt của bạn không hỗ trợ video này.</p>
 </video>
 
-> 💡 Gatsby uses **hot reloading** to speed up your development process. Essentially, when you’re running a Gatsby development server, the Gatsby site files are being “watched” in the background — any time you save a file, your changes will be immediately reflected in the browser. You don’t need to hard refresh the page or restart the development server — your changes just appear.
+> 💡 Gatsby sử dụng **hot reloading** để tăng tốc quá trình phát triển của bạn. Về cơ bản, khi bạn đang chạy máy chủ (server) phát triển Gatsbyto, các tệp của trang Gatsby được xem ở background - bất cứ khi nào bạn lưu tệp, các thay đổi của bạn sẽ lập tức được phản ánh trong trình duyệt. Bạn không cần phải làm mới trang hoặc khởi động lại máy chủ phát triển - các thay đổi của bạn mới xuất hiện.
```

Comment:
Gatsbyto - typo?
"watched" -> được theo dõi
in the background -> ở phía hậu trường, chú thích thêm (background) giúp mình nhá
your changes just appear -> các thay đổi của bạn sẽ xuất hiện (dịch vậy cho thuận miệng vậy...)